client/components/happiness-engineers-tray/style.scss

```
@@ -0,0 +1,12 @@
+.happiness-engineers-tray {
+	display: flex;
+	flex-wrap: wrap;
+	height: 42px;
+	justify-content: space-between;
+	margin-bottom: 16px;
```

Adding a negative margin right here should remove the extra right space we have now.
```suggestion
	margin-bottom: 16px;
	margin-right: -8px;
```